Fabric.Identity.API/scripts/Install-IdentityProviderSearchService.ps1

```diff
@@ -0,0 +1,61 @@
+﻿param(
+    [Parameter(Mandatory = $True)]
+    [PSCredential] $credential, 
+    [Hashtable] $configStore = @{Type = "File"; Format = "XML"; Path = "$PSScriptRoot\install.config"},
+    [Hashtable] $certificates,
+    [switch] $quiet
+)
+
+Import-Module -Name .\Install-Identity-Utilities.psm1 -Force
+
+# Get Idpss app pool user 
+# Create log directory with read/write permissions for app pool user
+# using methods in DosInstallUtilites to install idpss, which will make it easier to migrate the identity code later 
+$idpssSettingsScope = "identityProviderSearchService"
+$configStore = @{Type = "File"; Format = "XML"; Path = $installConfigPath}
```

Comment:
Should we only do this if $configStore is null? Looks like we default it...so this would overwrite it.

Reply:
I think I can just remove this since that was before I set a default on the parameter above, since it wouldn't know what $installConfigPath was.